.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -31,13 +31,21 @@ jobs:
         export WINDOWS_E2E_IMAGE=${{ parameters.windowsImage }}
         export WINDOWS_E2E_OSSKU=${{ parameters.windowsOSSKU }}
         export WINDOWS_E2E_VMSIZE=${{ parameters.windowsVMSize }}
-        export WINDOWS_E2E_STORAGE_ACCOUNT=${{ parameters.storageAccount }}
+        export windowsPackageURL="https://acs-mirror.azureedge.net/kubernetes/v${KUBERNETES_VERSION}/windowszip/v${KUBERNETES_VERSION}-1int.zip"
+        if [[ -n "$WINDOWS_PACKAGE_URL" ]]; then
+          export windowsPackageURL="$WINDOWS_PACKAGE_URL"
+        fi
+        export WINDOWS_PACKAGE_VERSION=$(echo $windowsPackageURL | grep -oP '(?<=/v)\d+\.\d+\.\d+' | head -1)
```

Comment:
I think that we should not use the version from the package url. By design, the version in the package url should always match the k8s version set in the pipeline variable. 
I think that we cannot test the hotfix without a workaround since all supported k8s versions do not have a hotfix package. It can be ignored since you have validated 1.25.2-hotfix package

Reply:
Are you saying we should just provide the k8s version set in the pipeline variable and remove the package url variable?

Or you are suggesting keeping the url variable 

 `$KUBENERTES_VERSION=1.24.9`, package = 1.25.2-hotfix (*workaround??*)

Reply:
Added check on the versions before setting up the test cluster. Windows package version (extracted from the non-empty) url should be the same as the KUBERNETES_VERSION pipeline variable.